src/main/scala/com/microsoft/hyperspace/actions/RefreshActionBase.scala

```diff
@@ -86,7 +87,10 @@ private[actions] abstract class RefreshActionBase(
 
   protected lazy val indexConfig: IndexConfig = {
     val ddColumns = previousIndexLogEntry.derivedDataset.properties.columns
-    IndexConfig(previousIndexLogEntry.name, ddColumns.indexed, ddColumns.included)
+    IndexConfig(
+      previousIndexLogEntry.name,
+      SchemaUtils.removePrefixNestedFieldNames(ddColumns.indexed),
```

Comment:
We remove the prefixes as based on these details we later prepare the dataset inside the `prepareIndexDataFrame`.